test/strscan/test_stringscanner.rb

```diff
@@ -783,6 +783,14 @@ def test_scan_aref_repeatedly
     assert_equal "t",      s[1]
     assert_equal "ring",   s[2]
   end
+  
+  def test_named_captures
```

Comment:
We need to omit this test on non-CRuby:

```suggestion
  def test_named_captures
    omit("not implemented on JRuby and TruffleRuby") if ["jruby", "truffleruby"].include?(RUBY_ENGINE)
```

Reply:
I understand the reason why some tests are not passed. It is available only for CRuby,